crates/astria-composer/src/lib.rs

```diff
@@ -30,7 +30,7 @@
 //!     .expect("failed to setup telemetry");
 //! info!(config = cfg_ser, "initializing composer",);
 //!
-//! let _composer = Composer::from_config(&cfg)
+//! let _composer = Composer::from_config(&cfg).await
```

Comment:
Consistent doc formatting:
```suggestion
//! let _composer = Composer::from_config(&cfg)
//!     .await